.github/ISSUE_TEMPLATE/linkcheck_report.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 🐛 Weekly linkcheck failed
+about: Use this template for checking broken links
+name: Weekly linkcheck failed
+labels: maintenance, bug
+---
+
+Links are checked weekly to make sure the documentation links areg working. Last check on {{ date | date('dddd, MMM Do YYYY, hh:mm A') }} failed. 
+
+Partial logs output can be found here:
```

Comment:
I think either "partial log output" or "partial logs" (with no "output") would be a bit better here.
```suggestion
Partial logs can be found here:
```
In what sense are they partial?